api/v1alpha1/webhooks/author/webhook.go

```diff
@@ -125,20 +125,20 @@ func (a *authorWebhook) handleReleasePlan(req admission.Request) admission.Respo
 		if releasePlan.GetLabels()[metadata.AttributionLabel] == "true" {
 			author := releasePlan.GetLabels()[metadata.AuthorLabel]
 
-			if oldReleasePlan.GetLabels()[metadata.AttributionLabel] != "true" || author == a.sanitizeLabelValue(req.UserInfo.Username) {
-				a.setAuthorLabel(req.UserInfo.Username, releasePlan)
+			if oldReleasePlan.GetLabels()[metadata.AttributionLabel] != "true" || author == w.sanitizeLabelValue(req.UserInfo.Username) {
+				w.setAuthorLabel(req.UserInfo.Username, releasePlan)
```

Comment:
How is this change necessary for adding toolkit and not a rewrite?

Reply:
Consistency.